saleor/graphql/order/types.py

```diff
@@ -468,6 +515,12 @@ class Order(CountableDjangoObjectType):
         ),
     )
 
+    discounts = graphene.List(
```

Comment:
Could we add it to the existing benchmark or create a new benchmark for getting few orders with many discounts?